gatsby/gatsby-config.js

```diff
@@ -55,7 +55,7 @@ module.exports = {
         // GitHub Flavored Markdown mode (default: true)
         gfm: true,
         // Plugins configs
-        plugins: [],
+        plugins: [`gatsby-remark-responsive-iframe`],
```

Comment:
this sounds nice after reading about it. Didn't know this existed. Seems like its specific to usage within Markdown files so I'm not sure if our components/index.js file will get the benefit of using it or not.